src/Maatwebsite/Excel/Classes/LaravelExcelWorksheet.php

```diff
@@ -637,9 +630,9 @@ protected function addData($array)
 
         // Add results
         if (!empty($data))
-            $this->data = !empty($this->data) ? array_merge($this->data, $data) : $data;
+            return !empty($this->data) ? array_merge($this->data, $data) : $data;
```

Comment:
This doesn't change behaviour. Check line 555. It'll ether way assign to `$this->data`